Gopkg.toml

```diff
@@ -31,7 +31,8 @@
 
 [[constraint]]
   name = "github.com/blackducksoftware/perceptor"
-  branch = "master"
+  #version = "v2.2.2"
+  branch = "support-multiple-hubs"
```

Comment:
Should be master. :)

Reply:
Once the perceptor PR is merged, will change it @rrati

Reply:
Updated to master